assets/js/googlesitekit/datastore/user/disconnect.test.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * core/user Data store: disconnect tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createTestRegistry, unsubscribeFromAll } from '../../../../../tests/js/utils';
+import { STORE_NAME } from './constants';
+
+describe( 'core/user disconnect', () => {
+	let registry;
+	const coreUserDataDisconnectEndpointRegExp = /^\/google-site-kit\/v1\/core\/user\/data\/disconnect/;
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	describe( 'disconnect', () => {
+		// Create a mock to avoid triggering a network request error.
+		// The return value is irrelevant to the test.
+		fetch
+			.doMockOnceIf( coreUserDataDisconnectEndpointRegExp )
+			.mockResponseOnce(
+				JSON.stringify( {} ),
+				{ status: 200 }
+			);
```

Comment:
Let's move this within the test that triggers it below. I think this probably works as-is but let's avoid adding code in the body of a `describe` like this and put it either in an individual test or in a `before(All|Each)` function.